scim_rails.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.required_ruby_version = "~> 2.4"
```

Comment:
We should probably use a newer version of ruby that is not at end of life.
https://endoflife.date/ruby